.drone.yml

```diff
@@ -432,43 +432,20 @@ steps:
     event:
     - tag
 
-- name: github_binary_prerelease
-  image: plugins/github-release
-  settings:
-    api_key:
-      from_secret: github_token
-    checksum:
-    - sha256
-    files:
-    - "bin/rancher-*"
-    prerelease: true
-    title: "Pre-release ${DRONE_TAG}"
-    note: ./bin/rancher-components.txt
-    overwrite: true
-  when:
-    event:
-    - tag
-    ref:
-      include:
-      - "refs/tags/*-*"
-
-- name: github_binary_release
-  image: plugins/github-release
+- name: artifacts-upload
+  image: plugins/s3
   settings:
-    api_key:
-      from_secret: github_token
-    checksum:
-    - sha256
-    files:
```

Comment:
Since we're no longer publishing the image lists to github, how are the `docker-image-digests-linux` and `docker-image-digests-windows` steps supposed to work going forward? 

Currently both of those steps  attempt to find the image list from the release artifacts on github. Without updating those steps we won't be able to generate the image digest list that is used to sync images to registries.

Reply:
References updated